tests/test_decorators.py

```diff
@@ -100,4 +100,4 @@ def test_csrf_rotation(self):
             lambda cls, root, info, *args, **kwargs: None,
         )(self, None, info_mock)
 
-        self.assertTrue(info_mock.context.csrf_cookie_needs_reset)
+        self.assertTrue(info_mock.context.META['CSRF_COOKIE_NEEDS_UPDATE'])
```

Comment:
I see double quotes being used in this library, maybe we should continue doing so?